app/src/main/java/vn/sun/music_68/ui/splashscreen/SplashActivity.java

```diff
@@ -0,0 +1,29 @@
+package vn.sun.music_68.ui.splashscreen;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import vn.sun.music_68.ui.main.MainActivity;
+
+public class SplashActivity extends AppCompatActivity {
+    private static final int SPLASH_DISPLAY_TIME = 2500;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        new Handler().postDelayed(new Runnable() {
```

Comment:
Sugget change
https://android.jlelse.eu/right-way-to-create-splash-screen-on-android-e7f1709ba154